plugins/source/aws/resources/services/efs/filesystems.go

```diff
@@ -82,3 +87,26 @@ func ResolveEfsFilesystemBackupPolicyStatus(ctx context.Context, meta schema.Cli
 
 	return resource.Set(c.Name, response.BackupPolicy.Status)
 }
+
+func ResolveEfsFilesystemPolicy(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
+	p := resource.Item.(types.FileSystemDescription)
+	config := efs.DescribeFileSystemPolicyInput{
+		FileSystemId: p.FileSystemId,
+	}
+	cl := meta.(*client.Client)
+	svc := cl.Services(client.AWSServiceEfs).Efs
+	response, err := svc.DescribeFileSystemPolicy(ctx, &config, func(options *efs.Options) {
+		options.Region = cl.Region
+	})
+	if err != nil {
+		if cl.IsNotFoundError(err) {
+			return resource.Set(c.Name, types.StatusDisabled)
```

Comment:
Instead of setting this to a value, can you just leave it unset to follow the convention of all of the other resources? Also a 404 doesn't automatically mean that it is disabled, the resource could have also been deleted in the time between the table resolver being called and the column resolver being called.